scripts/main.nut

```diff
@@ -58,4 +58,5 @@ scripts_load("module/ramp.nut")
 scripts_load("module/taxi_stunting.nut")
 scripts_load("module/admin.nut")
 //scripts_load("module/debug.nut")
+scripts_load("module/stadium.nut")
```

Comment:
nit: I'd move this script to the very top.
Generally, we should sort these scripts into different blocks.

The existing sorting was: 

- "base"
- "core functionality"
- "owner functionality".

The sorting was lost over time though.